src/map/status.cpp

```diff
@@ -14926,33 +14926,34 @@ void status_change_clear_buffs(struct block_list* bl, uint8 type)
 	for (const auto &it : status_db) {
 		sc_type status = static_cast<sc_type>(it.first);
 		std::bitset<SCF_MAX> flag = it.second->flag;
```

Comment:
While you're here, can you const ref this? We don't need to make a copy of the bitset.